### PR TITLE
New package: emilua-0.4.3

### PR DESCRIPTION
**Pull request by manipuladordedados**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/emilua/template [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Template file for 'emilua'
+pkgname=emilua
+version=0.4.3
+revision=1
+archs="x86_64"
+build_style=meson
+build_wrksrc="${pkgname}-v${version}"
+hostmakedepends="LuaJIT-devel boost-devel fmt-devel openssl-devel ncurses-devel serd-devel sord-devel liburing-devel libcap-devel"
+makedepends="meson boost re2c gawk gperf xxd ruby-asciidoctor pkg-config"
```
Comment (Duncaen) [inline srcpkgs/emilua/template:9]: `hostmakedepends` and `makedepends` are the opposite.

File: srcpkgs/emilua/template [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Template file for 'emilua'
+pkgname=emilua
+version=0.4.3
+revision=1
+archs="x86_64"
```
Comment (Duncaen) [inline srcpkgs/emilua/template:5]: This should not be limited to one architecture and libc unless there is a hard limitation.

File: srcpkgs/emilua/template [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+build_style=meson
+build_wrksrc="${pkgname}-v${version}"
```
Comment (Duncaen) [inline srcpkgs/emilua/template:7]:
```suggestion
build_wrksrc="emilua-v${version}"
```
the directory name is not really variable.

File: srcpkgs/emilua/template [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+c70033f50a067651943b15cd5fed0d407ab79bc2c686c68af796ace981a746ae"
+
+configure_args="
+	-Dversion_suffix=-void1 \
+	-Denable_http=true \
+	-Denable_file_io=true \
+	-Denable_io_uring=true \
+	-Denable_linux_namespaces=true \
+	-Denable_tests=true \
+	-Denable_manpages=true"
```
Comment (Duncaen) [inline srcpkgs/emilua/template:27]: Indent with a single space and remove the trailing backslashes, they are not necessary.

File: srcpkgs/emilua/template (hunk 1 above)
Comment (manipuladordedados) [reply srcpkgs/emilua/template:9]: Oh, my mistake.
I will update the template.

File: srcpkgs/emilua/template (hunk 2 above)
Comment (manipuladordedados) [reply srcpkgs/emilua/template]: Fixed it, only build for x86_64-glibc and aarch64 at the moment.

**Pull request closed without merging**